python/arcticdb/_msgpack_compat.py

```diff
@@ -4,15 +4,54 @@
 
 This module implements a backwards compatible version of msgpack functions.
 """
-
 import msgpack
+from arcticdb.log import version as log
+from arcticdb.exceptions import ArcticNativeException
+
+ExtType = msgpack.ExtType
+
+
+def _check_valid_msgpack():
+    pack_module = msgpack.packb.__module__
+    packer_module = msgpack.Packer.__module__
+    # Check that msgpack hasn't been monkey patched by another package
+    # We only support the official cmsgpack and fallback modules
+    if (pack_module in ("msgpack", "msgpack.fallback")) and (
+        packer_module in ("msgpack._packer", "msgpack.fallback", "msgpack._cmsgpack")
+    ):
+        return
+    log.info("Unsupported msgpack variant, got: {}, {}".format(pack_module, packer_module))
```

Comment:
Don't need this log if we're throwing anyway

Reply:
done